app/main.ipynb

```diff
@@ -493,26 +1686,74 @@
     "    result = total / len(lst)\n",
     "    return result\n",
     "\n",
-    "lst = [1, 2, 3, 4, 5]"
+    "lst = [1, 2, 3, 4, 5]\n",
+    "\n",
+    "%time unknown_signature(lst)\n",
+    "unknown_signature(lst)"
```

Comment:
The function `unknown_signature` calculates the mean of a list. Consider using NumPy's `np.mean` for better performance and simplicity.